src/cider/common/hashtable/FixedHashTable.h

```diff
@@ -147,12 +147,12 @@ class FixedHashTable : private boost::noncopyable,
   Cell* buf;  /// A piece of memory for all elements.
 
   void alloc() {
-    buf = reinterpret_cast<Cell*>(Allocator::allocate(NUM_CELLS * sizeof(Cell)));
+    buf = reinterpret_cast<Cell*>(Allocator::allocate(8 * NUM_CELLS * sizeof(Cell)));

```

Comment:
why hard code as 8?

Reply:
oh, thanks~ I forgot to remove this, it's for test only.